apps/jest-expo-mock-generator/App.js

```diff
@@ -74,7 +74,7 @@ export default class App extends React.Component {
     const moduleSpecs = await _getExpoModuleSpecsAsync();
     const code = `module.exports = ${JSON.stringify(moduleSpecs, replacer)};`;
     await setStringAsync(code);
-    this.setState({ moduleSpecs });
+    this.setState({ moduleSpecs: code });
```

Comment:
Set `moduleSpecs` as code because `setStringAsync` expects a string as the first parameter 

https://github.com/expo/expo/blob/78e08d0b6e13a32670db02c0a232803b9fc233f1/apps/jest-expo-mock-generator/App.js#L104